src/decode.h

```diff
@@ -1322,6 +1326,9 @@ void DecodeUnregisterCounters(void);
 #define PKT_FIRST_ALERTS BIT_U32(29)
 #define PKT_FIRST_TAG    BIT_U32(30)
 
+/** Unknown/unsupported */
+#define PKT_IS_UNKNOWN BIT_U32(31)
```

Comment:
I don't want to use this flag space for a protocol specific counter. Also, the flag is named way to generically.